src/main.rs

```diff
@@ -23,15 +27,18 @@ struct Opts {
 fn main() -> Result<()> {
     let opts = Opts::parse();
 
+    let index = opts.index;
+    let symbol = index.as_bitmex_symbol();
+
     let mut outcomes = Vec::new();
     for ResultsPage { count, start } in ResultsPages::new(opts.past_hours * 60).0.iter() {
         let mut url =
             reqwest::Url::parse("https://www.bitmex.com/api/v1/instrument/compositeIndex")?;
         url.query_pairs_mut()
-            .append_pair("symbol", ".BXBT") // only interested in index
+            .append_pair("symbol", &format!(".{symbol}")) // only interested in index
             .append_pair(
                 "filter",
-                r#"{"symbol": ".BXBT", "timestamp.ss": "00"}"#, // per minute
+                &format!("{{\"symbol\": \".{symbol}\", \"timestamp.ss\": \"00\"}}"), // per minute
```

Comment:
Does `format!` work with raw strings(`r#"..."`) ?

Then you wouldn't need to escape the quotes.

Reply:
I tried that at first and it didn't work :(